src/style/color-palette-extended.css

```diff
@@ -17,7 +25,6 @@
     --lime-brand-color-sellable-orange: 255, 112, 67; /* #ff7043 */
     --lime-brand-color-orange: 255, 176, 59; /* #ffb03b */
     --lime-brand-color-yellow: 255, 207, 61; /* #ffcf3d */
-    --lime-brand-color-lime-green: 102, 187, 106; /* #66bb6a */
```

Comment:
Urgh…

I can see arguments for keeping the old colours under the old names, and I can see arguments for updating the old colour variables to use the new colours instead too. But keeping all but one, which instead is updated, is kind of the worst of both options… 😣

Also, if we learn from this, and assume that every few years we'll get a new "brand refresh" with slightly tweaked colours… should we perhaps include a year in the colour names or something?

Reply:
I don't think we need the years. If the same name changes its value may be it.
if a Lime CRM Configurator person has chosen a brand color to paint something, they have to leave with their choice I'd say. In the CRM, we have a color picker that doesn't show any of these brand colors, and they cannot and should not use them really. If they do, they're to blame, not us. They have explicitly in that case said that they want whatever that brand color gives them, and don't care about UI or accessibility, etc…

Reply:
Ok 👍 🤷‍♂️ 😄

Reply:
Although, to be fair, what you said above sounds like a pretty good argument for just not including the brand colours to begin with… 🤔